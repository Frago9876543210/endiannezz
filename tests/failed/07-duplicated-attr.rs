use endiannezz::Io;

#[derive(Io)]
#[repr(C)]
#[endian(big)]
struct WorksFine;

#[derive(Io)]
#[endian(big)]
#[endian(native)]
struct Nope;

fn main() {}
