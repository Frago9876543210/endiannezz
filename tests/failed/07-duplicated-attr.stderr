error: please specify default endian
  --> $DIR/07-duplicated-attr.rs:9:1
   |
9  | / #[endian(big)]
10 | | #[endian(native)]
11 | | struct Nope;
   | |____________^
